data/web/inc/init_db.inc.php

```diff
@@ -3,7 +3,7 @@ function init_db_schema() {
   try {
     global $pdo;
 
-    $db_version = "26022024_1433";
+    $db_version = "26022024_1434";
```

Comment:
Please do the current time stamp eg. 13082024_0908

Its basically: ddmmyyyy_hhmm

Reply:
Done, it's set to 13082024_0927